### PR TITLE
Chore/28 env example and mock data

### PR DESCRIPTION
**Pull request by eduardomonforte**

### CODE REVIEW

**Comment by martiuh** [comment]
Remember to write closes #28 when you write a PR @eduardomonforte

File: .env.example [hunk 1]
```diff
@@ -1,13 +1,18 @@
 # JWT
+
 JWT_SECRET=*your data*
```
Comment (martiuh) [inline .env.example]: Change `*your data*` for `"your data"`

Comment (martiuh) [reply .env.example]: In every case

File: server/devBootScripts/initialData.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
+// const moment = require('moment');
```
Comment (martiuh) [inline server/devBootScripts/initialData.js]: Remove commented code

**Review by mishelashala — APPROVED** [review]

**Review by martiuh — APPROVED** [review]

**Review by jcomoto — APPROVED** [review]

**Pull request merged**